src/components/informational/QuerySpinner.wc.svelte

```diff
@@ -0,0 +1,51 @@
+<svelte:options
+    customElement={{
+        tag: "lens-query-spinner",
+    }}
+/>
+
+<script lang="ts">
+    import { responseStore } from "../../stores/response";
+    import type { ResponseStore } from "../../types/backend";
+
+    export let size = "20px";
+
+    let loading = false;
+
+    window.addEventListener("emit-lens-query", function () {
+        loading = true;
+    });
+
+    responseStore.subscribe((s: ResponseStore) => {
+        for (const value of s) {
+            if (value[1].status === "claimed") {
+                return;
+            }
+            loading = false;
+        }
+    });
```

Comment:
This logic means if any of the sites is not claimed, stop the spinner. Don't we want to wait for all of the sites to be not claimed until we stop the spinner?